src/Microsoft.SqlTools.ServiceLayer/Agent/AgentService.cs

```diff
@@ -1551,7 +1551,9 @@ string targetDatabase
                 DataTable materializedNotebookTable = await AgentNotebookHelper.GetAgentNotebookHistories(connInfo, jobId, targetDatabase);
                 foreach (DataRow materializedNotebookRow in materializedNotebookTable.Rows)
                 {
-                    string materializedRunDateTime = materializedNotebookRow["run_date"].ToString() + materializedNotebookRow["run_time"].ToString();
+                    // Adding a leading zero if the run time is before than 10am. Example: 9:10:00 will become 09:10:00 
+                    string notebookRuntime = (materializedNotebookRow["run_time"].ToString().Length == 5) ? "0" + materializedNotebookRow["run_time"].ToString() : materializedNotebookRow["run_time"].ToString();
```

Comment:
Your example doesn't match the logic here....`9:10:00` has 7 characters so wouldn't add the leading zero. Does it actually come in as `9:10`?

Reply:
Sorry, if I was not clear about it but it returned as 91000 and I am converting it to 091000.